todo_app/templates/index.html

```diff
@@ -12,7 +12,17 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+          <uL>
+            {% for item in items %}
+              <li>{{ item.id }} - {{ item.status }} - {{ item.title }}</li>
+            {% endfor %}
+          </uL>
       </ul>
     </div>
   </div>
+  <form action="/additem" method="POST">
+    <label for="nitem">New ToDo entry:</label><br>
+    <input type="text" id="nitem" name="nitem" value="Enter ToDo"><br><br>
```

Comment:
Generally not worth abbreviating names like this. Be clear rather than worrying about saving a few characters. E.g. `item_name` rather than `nitem`